autorest/utility.go

```diff
@@ -138,11 +138,31 @@ func MapToValues(m map[string]interface{}) url.Values {
 	return v
 }
 
+// ToStringSlice method converts interface{} to []string.
+func ToStringSlice(s interface{}) []string {
```

Comment:
I think this function should return an error when `s` isn't something convertible to a string slice.

Reply:
Agreed, kind of sucks that all the functions there don't follow this pattern. I created a proposal here for improving #211